cha-ching-app/src/SubscriptionButton.css

```diff
@@ -1,17 +1,15 @@
-.buttonsWrapper {
-  align-items: center;
-  display: flex;
-  height: 100vh;
-  justify-content: center;
-  width: 100vw;
-}
 
 .SubscriptionButton {
   border-radius: 20px;
   cursor: pointer;
   font-family: Arial, Helvetica, sans-serif;
   font-size: 22px;
   font-weight: 500;
-  height: 100px;
-  width: 200px;
+  height: 50px;
+  width: 180px;
+  background-color: rgb(253, 200, 206, 0.42);
```

Comment:
this section right here is almost perfectly alphebetized - background-color just needs to be moved to the top